src/subtitles/subtitles.js

```diff
@@ -3,6 +3,9 @@ import findSegmentTemplate from "../utils/findtemplate"
 
 function Subtitles(mediaPlayer, autoStart, playbackElement, defaultStyleOpts, mediaSources, callback) {
   const useLegacySubs = window.bigscreenPlayer?.overrides?.legacySubtitles ?? false
+  const isSeekableLiveSupport = window.bigscreenPlayer.liveSupport
```

Comment:
I had trouble following this – Could we separate it into two assertions like `is liveSupport nullish or equal to "seekable"`?